reactNativeExample/ios/reactNativeExample.xcodeproj/project.pbxproj

```diff
@@ -664,7 +663,7 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
+				PRODUCT_BUNDLE_IDENTIFIER = com.authgear.reactNativeExample;
```

Comment:
The bundle identifier should be in lowercase. I suggest using the ID `com.authgear.example.reactnative`.

Reply:
changed